tools/converter/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+add_subdirectory(binary_reader)
+# Convert binary reports to SONATA format
+add_executable(spikes_converter spikes_converter.cpp)
+target_include_directories(spikes_converter
+    PRIVATE
+        $<BUILD_INTERFACE:${SONATA_REPORT_INCLUDE_DIR}>
+        $<BUILD_INTERFACE:${SONATA_REPORT_SRC_DIR}>
+    SYSTEM
+        ${PROJECT_SOURCE_DIR}/extlib/spdlog/include
+)
+target_compile_options(spikes_converter
+    PRIVATE ${SONATA_REPORT_COMPILE_OPTIONS}
+)
+target_link_libraries(spikes_converter
+    PUBLIC sonata_report
+    PRIVATE spdlog::spdlog_header_only
+    PRIVATE ${MPI_CXX_LIBRARIES}
+)
```

Comment:
these tools require MPI linking because the libsonata is MPI enabled, is it?

Reply:
Yes, is needed but actually I was missing the MPI_Init/Finalize calls in case of parallel implementation (mpi+hdf5 parallel)

Reply:
By the way, IIRC, we should use MPI_C_LIBRARIES than MPI_CXX_LIBRARIES everywhere. (C++ part is removed from new MPI standard so doesn't make sense to link to C++ libs. We use just MPI C API)